content/zh/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -1,5 +1,6 @@
 ---
-title: Using a Service to Expose Your App
+<!--title: Using a Service to Expose Your App-->
+title: 使用 Service 暴露您的应用
 weight: 10
 ---
```

Comment:
Line 1-5 above is the "front-matter" for a content page. Please treat it as a single unit rather than insert comments into it. It may prove to work after all. Just want to be cautious at structuring things. A better option:
```
---
title: 中文翻译
weight: 10
---
<!--
title: english title
weight: 10
-->
```

Note that the *front-matter* must appear before the comment lines just so that the parser won't break accidentally.